src/Integration.Vsix/Settings/OtherOptionsDialogPage.cs

```diff
@@ -28,47 +29,50 @@ internal class OtherOptionsDialogPage : UIElementDialogPage
     {
         public const string PageName = "Other";
 
-        private readonly ITelemetryManager telemetryManager;
+        private ITelemetryManager telemetryManager;
 
         private OtherOptionsDialogControl optionsDialogControl;
         protected override UIElement Child => optionsDialogControl ?? (optionsDialogControl = new OtherOptionsDialogControl());
 
-        public OtherOptionsDialogPage()
```

Comment:
Similar to the _GeneralOptionsDialogPage_ - the page has access to a service provider via the _Site_ property so it doesn't need to use the _GlobalProvider._ 

I did consider leaving the constructor in line 41 as it makes testing easier, but decided to remove it. It makes the test setup more complicated, but it means the tests are covering the actual code paths used at runtime.